autoscheduler/frontend/src/components/Schedule/Schedule.tsx

```diff
@@ -281,7 +280,7 @@ const Schedule: React.FC<RouteComponentProps> = () => {
         availability={availability}
         firstHour={FIRST_HOUR}
         lastHour={LAST_HOUR}
-        key={`${availability.startTimeHours}:${availability.startTimeMinutes}`}
+        key={formatTime(availability.startTimeHours, availability.startTimeMinutes, true)}
```

Comment:
```suggestion
        key={formatTime(availability.startTimeHours, availability.startTimeMinutes)}
```

Per my previous comment, this is now redundant.